src/routes/(infoPages)/layout.scss

```diff
@@ -1,3 +1,4 @@
 .top-tier-page-hero-image {
-  height: 33vh;
+  height: 14rem;
```

Comment:
This is definitely much cleaner, but the images can look overly-cropped at larger screen sizes. If we increase this value based on screen size (with static values, not `%`, `vh`, etc.) would that mess up any of the optimizations? I'm thinking maybe we increase it to `16rem` on `tablet` and then `18rem` on `desktop`?

Reply:
Sure, I'll look into this!